adapters/rest-adapter-generator/src/test/java/uk/gov/justice/services/adapters/rest/generator/RestAdapterGenerator_MediaTypeMapperTest.java

```diff
@@ -0,0 +1,122 @@
+package uk.gov.justice.services.adapters.rest.generator;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.not;
+import static org.hamcrest.CoreMatchers.nullValue;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.raml.model.ActionType.GET;
+import static org.raml.model.ActionType.POST;
+import static uk.gov.justice.services.adapters.test.utils.builder.HeadersBuilder.headersWith;
+import static uk.gov.justice.services.adapters.test.utils.builder.HttpActionBuilder.httpActionWithNoMapping;
+import static uk.gov.justice.services.adapters.test.utils.builder.RamlBuilder.restRamlWithDefaults;
+import static uk.gov.justice.services.adapters.test.utils.builder.ResourceBuilder.resource;
+import static uk.gov.justice.services.adapters.test.utils.config.GeneratorConfigUtil.configurationWithBasePackage;
+import static uk.gov.justice.services.adapters.test.utils.reflection.ReflectionUtil.methodOf;
+
+import java.lang.reflect.Method;
+import java.util.Map;
+
+import javax.inject.Named;
+
+import com.google.common.collect.ImmutableMap;
+import org.hamcrest.Matchers;
+import org.junit.Test;
+
+public class RestAdapterGenerator_MediaTypeMapperTest extends BaseRestAdapterGeneratorTest {
+
+    private static final Map<String, String> ACTION_MAPPING_FALSE = ImmutableMap.of("actionMapping", "false");
```

Comment:
This immutableMap seems common across multiple tests (true/false versions), can it be moved into the base test as protected?